corehq/pillows/user.py

```diff
@@ -69,13 +69,14 @@ def get_unknown_users_pillow(pillow_id='unknown-users-pillow'):
         pillow_id,
     )
     processor = UnknownUsersProcessor()
+    change_feed = KafkaChangeFeed(topics=[FORM, FORM_SQL], group_id='unknown-users')
     return ConstructedPillow(
         name=pillow_id,
         checkpoint=checkpoint,
-        change_feed=KafkaChangeFeed(topics=[FORM, FORM_SQL], group_id='unknown-users'),
+        change_feed=change_feed,
         processor=processor,
-        change_processed_event_handler=PillowCheckpointEventHandler(
-            checkpoint=checkpoint, checkpoint_frequency=100,
+        change_processed_event_handler=MultiTopicCheckpointEventHandler(
```

Comment:
(no need to from my side if that's the failure mode - i was worried they were flat out broken)